tests/test_other.py

```diff
@@ -200,7 +200,7 @@ def test_emcc_1(self):
 
       # -dumpversion
       output = run_process([PYTHON, compiler, '-dumpversion'], stdout=PIPE, stderr=PIPE)
-      self.assertEqual(shared.EMSCRIPTEN_VERSION + os.linesep, output.stdout, 'results should be identical')
+      self.assertEqual(shared.EMSCRIPTEN_VERSION, output.stdout.rstrip(), 'results should be identical')
```

Comment:
You probably want to drop the third argument too, so that the error shows the differing values.